modules/newrelic-http-exporter/src/main/scala/io/janstenpickle/trace4cats/newrelic/NewRelicSpanCompleter.scala

```diff
@@ -10,27 +10,29 @@ import io.janstenpickle.trace4cats.model.TraceProcess
 import org.http4s.client.Client
 import org.http4s.client.blaze.BlazeClientBuilder
 
+import scala.concurrent.ExecutionContext
+
 object NewRelicSpanCompleter {
-  def blazeClient[F[_]: ConcurrentEffect: Timer](
-    blocker: Blocker,
+  def blazeClient[F[_]: Async](
+    ec: ExecutionContext, //TODO: keep parameter or replace with EC.global as recommended?
```

Comment:
Here and everywhere:
https://gitter.im/http4s/http4s?at=603fd64f457d6b4a949ed0fe

There's no reason for using manually provided EC.